modules/ob-installing-builds-with-operator.adoc

```diff
@@ -0,0 +1,34 @@
+// This module is included in the following assembly:
+//
+// builds/installing-openshift-builds.adoc
+
+:_content-type: PROCEDURE
+[id="installing-builds-with-operator_{context}"]
+= Installing {builds-shortname} by using the Operator
+
+You can install the {builds-title} Operator from the OperatorHub by using the {product-title} web console. Installing this Operator enables you to install and use build components.
```

Comment:
Similar concern here. It would fit better if we have the `product-title` as `Red Hat OpenShift`.